apps/settings/locales/settings.en-US.properties

```diff
@@ -1356,6 +1356,9 @@ screenReader-toggle={{screenReader}}
 colorFilter=Color Filter
 colorFilter-header={{colorFilter}}
 colorFilter-toggle={{colorFilter}}
+monoAudio=Mono Audio
```

Comment:
Just have one entry instead of 3 and reuse it in both places:
accessibility-audio = Audio